cli/src/commands/extensions/permissions.rs

```diff
@@ -261,6 +261,16 @@ impl From<&Permissions> for PermissionsOptions {
             allow_sys: None,
             allow_ffi: None,
             allow_hrtime: false,
+
+            deny_read: None,
+            deny_write: None,
+            deny_run: None,
+            deny_env: None,
+            deny_net: None,
+            deny_sys: None,
+            deny_ffi: None,
+            deny_hrtime: false,
+
```

Comment:
What is their relation to the other permissions? Are they additional or will the `allow_` permissions be removed in the future?

Reply:
You can see the change in `deno` at denoland/deno@6fb7e8d9

The `deny_` permissions modify the `allow_` permissions so that a subset of what is allowed can be denied (e.g., `--allow-read=/etc --deny-read=/etc/passwd`)

I have not seen any intention to remove `allow_` permissions in the future.